docs/references/api/resource_embedding.rst

```diff
@@ -1042,6 +1042,10 @@ Both ``is.null`` and ``not.is.null`` can be included inside the `or` operator. F
     -d select=title,actors(*),directors(*)" \
     -d "or=(actors.is.null,directors.is.null)"
 
+.. note::
+  
+  PostgREST also provides ``is.not_null`` filter which works the same as ``not.is.null``. See `#3747 <https://github.com/PostgREST/postgrest/issues/3747>`_.
+
```

Comment:
```suggestion
```

I think we should only document this on https://github.com/PostgREST/postgrest/blob/60d92f64dcab681ffc1674cb87935cb0a638603b/docs/references/api/tables_views.rst#L75